gamification-crowdin-services/src/main/java/io/meeds/gamification/crowdin/plugin/SuggestionAddedTriggerPlugin.java

```diff
@@ -32,23 +31,22 @@ public void initData() {
 
     @Override
     public List<Event> getEvents(String trigger, Map<String, Object> payload, Object object) {
-        if (extractSubItem(payload, getPayloadObjectName(), "provider") != null) {
-            LOG.warn("Crowdin event {} translation provider is TM or MT", EVENT_TRIGGER);
-            return Collections.emptyList();
-        }
 
-        return Collections.singletonList(new Event(EVENT_TITLE,
-                extractSubItem(payload, getPayloadObjectName(), "user", "username"),
-                extractSubItem(payload, getPayloadObjectName(), "user", "username"),
-                extractSubItem(payload, getPayloadObjectName(), "string", "url"),
+        return Collections.singletonList(new Event(SUGGESTION_ADDED_EVENT_NAME,
+                extractSubItem(payload, EVENT_PAYLOAD_OBJECT_NAME, "user", "username"),
+                extractSubItem(payload, EVENT_PAYLOAD_OBJECT_NAME, "user", "username"),
```

Comment:
duplicate ?

Reply:
> duplicate ?

Nope, one is a sender, the other is a receiver. The event needs both, they do the same in the Github connector : 

Probably you can tell the team to help clarify in the notes,
https://github.com/Meeds-io/gamification-github/blob/cad434e2a854cd9f54abcb894aaf7b376480673f/gamification-github-services/src/main/java/io/meeds/gamification/github/plugin/PullRequestReviewTriggerPlugin.java#L35